tests/behaviour/contracts/conversions/storageArrays.sol

```diff
@@ -0,0 +1,102 @@
+pragma solidity ^0.8.10;
+
+//SPDX-License-Identifier: MIT
+
+contract WARP {
+    uint[] x;
+    uint[5] y;
+    function tryX1(uint8[] memory m) public returns (uint[] memory) {
```

Comment:
Worth adding tests that use int here as well as uint, as a conversion function is needed to widen ints, even when they're both felts